src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductVariantRepository.php

```diff
@@ -98,9 +98,19 @@ public function findByCodesAndProductCode(array $codes, string $productCode): ar
         ;
     }
 
-    public function findByCodes(array $codes): array
+    public function findByCodesWithProductCannelPricingsAndTaxons(array $codes): array
```

Comment:
```suggestion
    public function findByCodesWithProductChannelPricingsAndTaxons(array $codes): array
```
or even leave the old name 🤔 :
```suggestion
    public function findByCodes(array $codes): array
```